taskcluster/ac_taskgraph/build_config.py

```diff
@@ -27,7 +29,7 @@ def get_components():
 
 
 def get_version():
-    return _read_build_config()["componentsVersion"]
+    return ensure_text(_read_build_config()["componentsVersion"])
```

Comment:
I remember we talked about it on chat.mozilla.org. For the record, in case someone reads this comment later, `ensure_text()` is a good solution to make sure we deal with unicode strings no matter whether we're on py2 or py3.

https://six.readthedocs.io/#six.ensure_text
https://six.readthedocs.io/#six.text_type